scripts/odc-product-archive/archive_odc_product.py

```diff
@@ -41,8 +41,12 @@
     print("No datasets to archive................................................")
 else:
     df = pd.DataFrame(dataset_ids)
-    s3.Bucket("s3://deafrica-landsat/status-report/archived/").upload_file(
-        Filename=PRODUCT_NAME + "_archived.csv", Key=PRODUCT_NAME + "_archived.csv"
+    df.to_csv(PRODUCT_NAME + "_archived.csv")
+
+    # Upload to AWS s3 bucket
+    s3.Bucket("archive-deafrica-datasets").upload_file(
+        Filename=PRODUCT_NAME + "_archived.csv",
+        Key="crop-mask/" + PRODUCT_NAME + "_archived.csv",
```

Comment:
How can we avoid the specific reference to "crop-mask"?

Reply:
To avoid referencing to the folder "crop-mask", key must include the file name only. For example in this case :  if 
        Key=PRODUCT_NAME + "_archived.csv" , the file will uploaded to the root folder of the S3 bucket. The reason why I created the crop-mask folder is to store related  product in one folder.